### ISSUE TITLE
cffi version error in Vagrant image

### ISSUE THREAD

**Issue by Soft** [KVM-VMI/kvm-vmi#17]
I think there might be a version mismatch between the Python cffi version installed by the vagrant image and what the code in Nitro repository expects. When trying to run `vagrant up` provisioning fails with

```json
fatal: [default]: FAILED! => {
    "changed": true, 
    "cmd": [
        "python3", 
        "nitro/build_libvmi.py"
    ], 
    "delta": "0:00:00.043156", 
    "end": "2017-11-14 11:42:39.246956", 
    "failed": true, 
    "invocation": {
        "module_args": {
            "_raw_params": "python3 nitro/build_libvmi.py", 
            "_uses_shell": false, 
            "chdir": "/data/kvm-vmi/nitro", 
            "creates": null, 
            "executable": null, 
            "removes": null, 
            "stdin": null, 
            "warn": true
        }
    }, 
    "msg": "non-zero return code", 
    "rc": 1, 
    "start": "2017-11-14 11:42:39.203800", 
    "stderr": "Traceback (most recent call last):\n  File \"nitro/build_libvmi.py\", line 192, in <module>\n    ffibuilder.set_source(\"nitro._libvmi\",\nAttributeError: 'FFI' object has no attribute 'set_source'", 
    "stderr_lines": [
        "Traceback (most recent call last):", 
        "  File \"nitro/build_libvmi.py\", line 192, in <module>", 
        "    ffibuilder.set_source(\"nitro._libvmi\",", 
        "AttributeError: 'FFI' object has no attribute 'set_source'"
    ], 
    "stdout": "", 
    "stdout_lines": []
}
```

**Comment by Wenzel** [comment]
hey,

it should not call `set_source` for sure, because this API is only available with `CFFI` > 1.0.0

The latest `build_libvmi.py` uses `verify()`, which is a legacy API, but it allows use to use the debian jessie version of `python3-cffi`
https://github.com/KVM-VMI/nitro/blob/master/nitro/build_libvmi.py#L184

What i think is that you used vagrant without NFS, therefore, it cloned all the kvm-vmi repos, and since i didn't updated the submodules, you have an old version of Nitro in the vagrant box :)

Is that the case ?

**Comment by Soft** [comment]
You are right, I disabled NFS for the build.
